crates/librqbit/src/tracing_subscriber_config_utils.rs

```diff
@@ -107,7 +108,25 @@ pub fn init_logging(opts: InitLoggingOptions) -> anyhow::Result<InitLoggingResul
                     .with_writer(log_file)
                     .with_filter(
                         EnvFilter::builder()
-                            .parse(opts.log_file_rust_log.unwrap_or("info,librqbit=debug"))
+                            .parse(opts.log_file_rust_log)
+                            .context("can't parse log-file-rust-log")?,
+                    ),
+            )
+            .try_init()
+            .context("can't init logging")?;
+    } else if opts.structured_stdout {
```

Comment:
Why is it either logfile or structured stdout? Why not support both?

Reply:
If this is because of typing looks like you can use [```Option<Layer>```](https://stackoverflow.com/questions/77607599/how-to-add-a-layer-to-tracing-subscribers-registry-based-on-a-condition)